### ISSUE TITLE
Add continentality index

### ISSUE THREAD

**Issue by JordiBolibar** [ODINN-SciML/MassBalanceMachine#17]
Add the continentality index `k` from [Conrad 1946](https://glossary.ametsoc.org/wiki/Continentality).

- This should be internally computed for each stake position following the equation.
- Then this should be added in the training dataset table and used for training. To be checked that it doesn't lead to overfitting. 

We could also check this paper to better understand the relationship with glacier MB: https://www.cambridge.org/core/journals/annals-of-glaciology/article/effect-of-continentality-on-glacier-response-and-mass-balance/E89FA0BE224EAFB7B303D468B27E122A

**Comment by khsjursen** [comment]
Closing this issue as it is generally agreed upon that we don't want to rely on latitude as a feature since it potentially masks climate signals. Continentality index uses latitude in the calculation.
